src/test/scala/temple/generate/database/package.scala

```diff
@@ -170,5 +170,22 @@ package object database {
     val postgresInsertString: String =
       """INSERT INTO Users (id, bankBalance, name, isStudent, dateOfBirth, timeOfDay, expiry)
         |VALUES ($1, $2, $3, $4, $5, $6, $7);""".stripMargin
+
+    val deleteStatement = Delete(
+      "Users"
+    )
+
+    val postgresDeleteString: String =
+      """DELETE FROM Users;"""
+
+    val deleteStatementWithWhere = Delete(
```

Comment:
Are you gonna write a test to do the DELETE with WHERE checking?

Reply:
Will accept when complete